### ISSUE TITLE
Chem Recipe Dupe/Unbalanced Bugs 2.1.2.4+

### ISSUE THREAD

**Issue by OrderedSet86** [GTNewHorizons/GT-New-Horizons-Modpack#10484]
### Your GTNH Discord Username

Order#0001

### Your Pack Version

2.1.2.4-4

### Your Server

Private

### Type of Server

Vanilla Forge

# Your Expectation

I really like the chem formula balancing changes. However, they are incomplete and allow for lots of dupe bugs at the moment. I will try and list all them in the section below so they can hopefully be fixed (and the chem balancing changes kept).

# The Reality

(this is a living document, I'll still editing it and regularly saving it as I find new stuff)

### 1. Salt dupe 
Na + H2O -> NaH2O 
NaH2O -> 2Na + 2H2O???
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10477

### 2. Acetone-based oxygen, hydrogen, and carbon dupe 
2C + 4H + 2O -> C2H4O2
4C2H4O2 + Ca -> 4C3H6O + 4CO2 + 8H??? (+8C, +16H, +4O, Ca does not get consumed)
![image](https://user-images.githubusercontent.com/88752681/173462041-fb4f016c-6f2a-424e-8b74-78bf74ccbbe0.png)
![acetone](https://user-images.githubusercontent.com/88752681/173460226-10b80598-cca5-48d6-9c4f-d01610560ca1.png)

### 3. Clay compound (turns flint into clay and stone)
![image](https://user-images.githubusercontent.com/88752681/173460636-ec686a48-9bce-45de-9325-7f28bf182b62.png)
![image](https://user-images.githubusercontent.com/88752681/173460655-8a637401-4a84-4ac1-81f2-d7d7c87bc61d.png)

### 4. Clay plate based clay dupe (this is an old one, but should be fixed anyway)
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/10446

### 5. Cactus charcoal oxygen
Note that this isn't very viable at the moment, I'm mostly mentioning in case cactus charcoal gets a pyro/GT++ coke oven recipe. Right now this requires RC coke ovens.
![image](https://user-images.githubusercontent.com/88752681/173461244-12f6ef33-7f82-4c8f-a544-5c4f2e824f10.png)
![image](https://user-images.githubusercontent.com/88752681/173461271-9bbb7b1a-39d7-48ac-b9f9-2c01776411ec.png)

### 6. Absurdly expensive larger lava centrifuging recipe
tl;dr the recipe outputs are fine, but it costs 3x more EU/t and is slower at the same voltage
![image](https://user-images.githubusercontent.com/88752681/173464822-683c3631-f1fe-4683-8feb-e7ca0b6de06c.png)
![image](https://user-images.githubusercontent.com/88752681/173464831-cbc8eed3-dc01-4612-ac92-376384422212.png)
![lavalow](https://user-images.githubusercontent.com/88752681/173464845-3a01c21b-31aa-4624-acef-15b94d109e84.png)
![lavahigh](https://user-images.githubusercontent.com/88752681/173464852-4d691836-853c-41d0-af6a-7155f4cf6a07.png)

# Your Proposal

Make chem formulas fully balanced, not just partially.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by GlodBlock** [comment]
the chemical rebalance pr is still draft now. but dream directly merged it in the dev branch

**Comment by Dream-Master** [comment]
Yes I saw this and will do a version without draft prs. Inside sourcetree I can see only the branches and don’t see the prs

**Comment by chochem** [comment]
> the chemical rebalance pr is still draft now. but dream directly merged it in the dev branch

yeah, OrderedSet knows that. But this should help fix that PR so that it eventually can be merged fully.

**Comment by miozune** [comment]
Closing as all issues other than point 4 are fixed
